package.json

```diff
@@ -29,7 +29,7 @@
     "bitcore-lib": "0.14.0",
     "bitcore-lib-cash": "https://github.com/Bitcoin-com/bitcore-lib-cash.git",
     "bitcore-mnemonic": "1.2.2",
-    "bitcore-payment-protocol": "https://github.com/Bitcoin-com/bitcore-payment-protocol.git",
```

Comment:
Needs to link to the updated bitcoin.com repo once the changes have been made to it.

Reply:
Ready to do that whenever the PR is done on bitcore-payment-protocol